services/QuillLMS/app/views/pages/shared/_teacher_center_navbar.html.erb

```diff
@@ -1,75 +1,23 @@
-<% @all = 'All' %>
-<% @research = 'Research' %>
-<% @faq = 'FAQ' %>
-<% @premium = 'Premium' %>
+<%-# locals: active_tab %>
```

Comment:
Why is this here?

Reply:
It's a comment about local vars for this partial. I'm going to reword the comment to make that clearer. It's trying to treat the partial like a function and list its params.